### ISSUE TITLE
API access for VC funded startup

### ISSUE THREAD

**Issue by bipedal103** [soundcloud/api#240]
Hi

Excitr is a startup that aims to deepen the artist - fan connection. 
We are building features for fandom, such as rewarding fans, offering exclusive content and similar. 

We support Spotify and Deezer, and would like to add SoundCloud. 

We are VC funded and in active development. 

Is it possible to get access to the API in order to implement following of artists on SoundCloud, playing of tracks and Connect with SoundCloud feature?

Thank you. 

## Title: Cannot retrieve all tracks for a user
Issue found of: September 20th, 2020

### Endpoint(s):
* `GET /users/{id}/tracks`

### Scope(s):
* None (application is not using authentication i.e., implicit flow)

### Steps to reproduce:
```bash
curl --request GET \
  --url 'https://api.soundcloud.com/users/12345/tracks?limit=1&linked_partitioning=true&client_id=DONT_SHARE_CLIENT_ID'
```

### Expected behaviour:
A list of tracks as described at `https://developers.soundcloud.com/docs/api/reference#users`.

### Actual behaviour:
500 Server Error (consistently)

**Comment by youssefhassan** [comment]
Hey @bipedal103, I'm wondering if your request is still valid and whether you have a client_id already as I saw in the request you are using one

**Comment by bipedal103** [comment]
Hi @youssefhassan 
Thanks for following up on this. 
No, we don't have our own client id.
Should I open a new ticket and send you the id here?

Thank you
✌️

**Comment by MariaVWSoundCloud** [comment]
@bipedal103 Yes, please reach out to SoundCloud support (even through the chat bot) and ask for api access

**Comment by bipedal103** [comment]
@MariaVWSoundCloud appreciate the quick reply. 
My ticket number is 3191270, if needed. 

Thank you ✌️

**Comment by bipedal103** [comment]
@MariaVWSoundCloud just checking up on this. 
Best
